contribute/architecture/backend/services.md

```diff
@@ -107,31 +107,31 @@ func InitializeForTest(cla setting.CommandLineArgs, opts Options, apiOpts api.Se
 
 ## Background services
 
-A background service is a service that runs in the background of the lifecycle between Grafana starts up and shutdown. If you want a service to be run in the background your Service should satisfy the `registry.BackgroundService` interface and add it as argument to the [ProvideBackgroundServiceRegistry](/pkg/server/backgroundsvcs/background_services.go) function and add it as argument to `NewBackgroundServiceRegistry` to register it as a background service.
+A background service is a service that runs in the background of the lifecycle between Grafana startup and shutdown. If you want a service to be run in the background, your Service should satisfy the `registry.BackgroundService` interface and add it as argument to the [ProvideBackgroundServiceRegistry](/pkg/server/backgroundsvcs/background_services.go) function and add it as argument to `NewBackgroundServiceRegistry` to register it as a background service.
```

Comment:
I'm not sure about the word "satisfy" --is there another word we can use? I think the last sentence has double info-can you take another look?

Reply:
"Satisfy an interface" is a little odd, yes; usually you "implement" an interface. I just didn't touch that because I wasn't 100% sure what needs to happen, and/or if there's some special language around golang (which I don't know, personally).

Re-checking

Reply:
Looking around at Go language documentation -- "satisfy" is standard wording, so I'll keep that here.

An interface is basically a list of functions (sometimes a list of just one). If a service "satisfies" the interface, that means it passes the check "does it include all of the functions listed in the interface?"

Reply:
Ok:)